packages/components/src/toolbar/index.js

```diff
@@ -54,11 +56,12 @@ function Toolbar( { controls = [], children, className } ) {
 	}
 
 	return (
-		<div className={ classnames( 'components-toolbar', className ) }>
+		<ToolbarContainer className={ classnames( 'components-toolbar', className ) }>
```

Comment:
Can we convert `div` into `View` abstract component instead of reimplementing every `div` as a separate component?

Reply:
👋 @gziolo, I deliberately avoided subplanting `div` with `View` because it would move the whole problem of mirroring a particular `div`'s behavior to SCSS, and I don't think we should do that yet. SCSS support is still highly experimental in the RN app. So instead, I opted for implementing super specific components that are well defined in behavior, will be easy to reason about and will help us move along.

I actually expect patterns to start emerging the more `div`s we implement like that, and we'll start consolidating them into more general implementations, or better, indeed move them to general View+SCSS.

Reply:
Can you create follow up issue and leave inline comments so we make sure we clean it up when those patterns are established?

Reply:
A follow-up issue is fine, too. We can handle it, no worries @hypest.